src/view/SmartLogIn.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import userModel from "../model/UserModel";
+import Login from "./Login";
+import loginPreseneter from "../presenter/LoginPresenter";
```

Comment:
Also, typo in the Presenter word for the import variable (also copy-pasted around a little).